dace/transformation/interstate/sdfg_nesting.py

```diff
@@ -72,9 +72,17 @@ def _check_strides(inner_strides: List[symbolic.SymbolicType],
         ostrides = [
             os for i, os in enumerate(outer_strides) if i not in dims_to_ignore
         ]
+
+        istrides = [
+            os for i, os in enumerate(inner_strides) if i not in dims_to_ignore
+        ]
```

Comment:
the dims to ignore are for the outer memlet. I don't see how this works here

Reply:
Maybe I can do the  same, but considering  the inner memlet?